### PR TITLE
feat: added "fraction" statistic to the "analyze_var_count" method group

### PR DESCRIPTION
**Pull request by kpagacz**
Hello,

I wanted to display the count statistic in roughly this format: `xx / xx (xx.xx%)`, but it's impossible to use `analyze_vars` in this manner. I could implement my own `analyze_vars` and the associated functions, but I only need this slight change in the code, and copying the whole thing seems like an overkill.

Let me know if there's a more straightforward solution to display these statistics using the existing API (and not lose the S3 dispatch on different types of input variables).

@shajoezhu @pawelru @gogonzo If you can point me to the correct person to approach regarding this, I'd be grateful.

### CODE REVIEW

**Comment by kpagacz** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by pawelru** [comment]
@kpagacz please add NEWS entry and fix the tests. Please ignore CLA bot for a while - it's a known issue.

@insightsengineering/nest-sme please kindly help with the review of this

**Review by Melkiades — APPROVED** [review]
lgtm! As mentioned, add news and fix tests. Thanks @kpagacz!!

File: NEWS.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 * Added the `riskdiff` argument to `tabulate_rsp_subgroups` and `tabulate_survival_subgroups` to allow users to add a risk difference table column, and function `control_riskdiff` to specify settings for the risk difference column.
 * Added warning to `tabulate_rsp_subgroups` when `pval` statistic is selected but `df` has not been correctly generated to add p-values to the output table.
 * Added `n_rate` statistic as a non-default option to `estimate_incidence_rate` which returns both number of events observed and estimated incidence rate.
+* Added `fraction` statistic to the `analyze_var_count` method group.
+
```
Comment (kpagacz) [inline NEWS.md:10]:
```suggestion
```

**Pull request merged**